pubsub/google/cloud/pubsub_v1/subscriber/policy/thread.py

```diff
@@ -106,7 +106,10 @@ def __init__(self, client, subscription, flow_control=types.FlowControl(),
         # Also maintain a request queue and an executor.
         _LOGGER.debug('Creating callback requests thread (not starting).')
         if executor is None:
-            executor = futures.ThreadPoolExecutor(max_workers=10)
+            executor = futures.ThreadPoolExecutor(
+                max_workers=10,
+                thread_name_prefix='ThreadPoolExecutor-SubscriberPolicy',
```

Comment:
Original default is `''` which results in thread names like:

```
ThreadPoolExecutor-0_0
ThreadPoolExecutor-0_1
ThreadPoolExecutor-0_2
ThreadPoolExecutor-0_3
ThreadPoolExecutor-0_4
```

which AFAICT correspond to the prefix `ThreadPoolExecutor-0`.